src/core/obj.js

```diff
@@ -703,12 +703,18 @@ var Catalog = (function CatalogClosure() {
               'app.launchURL',
               'window.open'
             ];
-            var regex = new RegExp('^(?:' + URL_OPEN_METHODS.join('|') + ')' +
-                                   '\\((?:\'|\")(\\S+)(?:\'|\")(?:,|\\))');
+            var regex = new RegExp('^\\s*(' + URL_OPEN_METHODS.join('|') + ')' +
```

Comment:
Sure, however I believe that we'll need to use `.split('.').join('\\.')` since otherwise it won't make a difference for the resulting regexp.